run_gemini.py

```diff
@@ -0,0 +1,64 @@
+import google.generativeai as genai
+import os
+import subprocess
+import logging
+from dotenv import load_dotenv
+
```

Comment:
The variable `commit_branch` in `run_gemini.py` is not used in the code. This might be a leftover from previous development or a mistake.